pvlib/irradiance.py

```diff
@@ -1100,31 +1094,29 @@ def perez(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
     ebin = ebin.dropna().astype(int)
 
     # This is added because in cases where the sun is below the horizon
-    # (var.solar_zenith > 90) but there is still diffuse horizontal light
-    # (var.DHI>0), it is possible that the airmass (var.AM) could be NaN, which
-    # messes up later calculations. Instead, if the sun is down, and there is
-    # still var.DHI, we set the airmass to the airmass value on the horizon
-    # (approximately 37-38).
+    # (var.solar_zenith > 90) but there is still diffuse horizontal
+    # light (var.DHI>0), it is possible that the airmass (var.AM) could
+    # be NaN, which messes up later calculations. Instead, if the sun is
+    # down, and there is still var.DHI, we set the airmass to the
+    # airmass value on the horizon (approximately 37-38).
     # var.AM(var.solar_zenith >=90 & var.DHI >0) = 37;
 
     # var.DNI_ET[var.DNI_ET==0] = .00000001 #very hacky, fix this
 
     # delta is the sky's "brightness"
     delta = dhi * airmass / dni_extra
 
-    # keep only valid times
```

Comment:
the ebin.index changes from here to the end of the function are what really matters in this diff.